src/python/WMCore/ReqMgr/Service/Request.py

```diff
@@ -417,6 +415,11 @@ def _handleAssignmentStateTransition(self, workload, request_args, dn):
 
         # by default, it contains all unmerged LFNs (used by sites to protect the unmerged area)
         request_args['OutputModulesLFNBases'] = workload.listAllOutputModulesLFNBases()
+
+        # FIXME: remove it on HG1710, when this #7355 is complete fixed
```

Comment:
@ticoann Seangchan, as we discussed, I added this additional Team validation that we can remove in the next month or so